assembl/static2/js/app/components/administration/survey/mediaForm.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Translate, I18n } from 'react-redux-i18n';
+import { FormGroup, Checkbox, Button } from 'react-bootstrap';
+import { graphql, compose } from 'react-apollo';
+
+import FileUploader from '../../common/fileUploader';
+import uploadDocumentMutation from '../../../graphql/mutations/uploadDocument.graphql';
+
+import {
+  toggleMedia,
+  updateMediaHtmlCode,
+  updateMediaDescriptionTop,
+  updateMediaDescriptionBottom,
+  updateMediaDescriptionSide,
+  updateMediaTitle
+} from '../../../actions/adminActions';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+
+const relativeURL = (uRL) => {
+  return uRL.match(/^https?:\/\/.*?(\/.*)$/)[1];
```

Comment:
Why?

Reply:
Because we can upload a file and I needed to find a way to find out if it's an uploaded file or an external ressources to show or not the delete button and preview

Reply:
ok Now I understand

Reply:
You should create 2 input for each type: video and image. Then you don't need to do that.

Reply:
but there is only one backend field so I need this check anyway on subsequent reloads